### ISSUE TITLE
Large attachment upload blocks group chat messages from other senders

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#17394]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Create a group with few members
2. Send a large file attachment (20 MB size) from account A to the group
3. While the attachment is uploading, send multiple messages from account B to the group
4. On account A, go back from the group chat and open another chat
5. When the attachment upload is done, check the group chat on account A

## Expected Result:
The messages sent by account B should be displayed on account A’s group chat feed even if the attachment upload is not done. If account A opens another chat while uploading an attachment, the group chat should show an unread status indicator if new messages are sent by other accounts in the group.
## Actual Result:
The messages sent by account B are not displayed on account A’s group chat feed until the attachment upload is done. If account A opens another chat while uploading an attachment, the group chat does not show an unread status indicator even if new messages are sent by other accounts in the group. The messages appear only after account A opens the group chat again.

Additional Note: On Android devices, account A can receive new messages from account B in the group chat, but the unread status indicator is not displayed on the group chat.

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.0
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/43996225/231802218-887d67cb-c794-4f7e-8298-f32242838695.mp4

https://user-images.githubusercontent.com/43996225/231802243-4c6358e1-603f-4879-89f2-08b79e6336d6.mp4

**Expensify/Expensify Issue URL:**
**Issue reported by:** @nathan-mulugeta
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1681292087572289

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01f2cd61da9c999f71</li>
        <li>Upwork Job ID: 1648725726734278656</li>
        <li>Last Price Increase: 2023-04-19</li>
    </ul>
</details>

**Comment by MelvinBot** [comment]
### Bug0 Triage Checklist ([Main S/O](https://stackoverflowteams.com/c/expensify/questions/14418))

- [ ] This "bug" occurs on a supported platform (ensure `Platforms` in OP are ✅)
- [ ] This bug is not a duplicate report (check [E/App issues](https://github.com/Expensify/App/issues) and [#expensify-bugs](https://expensify.slack.com/archives/C049HHMV9SM))
    - If it is, comment with a link to the original report, close the issue and add any novel details to the original issue instead
- [ ] This bug is reproducible using the reproduction steps in the OP. [S/O](https://stackoverflowteams.com/c/expensify/questions/14418#:~:text=The%20bug%20is%20reproducible)
    - If the reproduction steps are clear and you're unable to reproduce the bug, check with the reporter and QA first, then close the issue.
    - If the reproduction steps aren't clear and you determine the correct steps, please update the OP.
- [ ] This issue is filled out as thoroughly and clearly as possible
    - Pay special attention to the title, results, platforms where the bug occurs, and if the bug happens on staging/production.
- [ ] I have reviewed and subscribed to the linked Slack conversation to ensure Slack/Github stay in sync

**Comment by michaelhaxhiu** [comment]
Ok I reproduced this after a little tinkering. Used the same attachment as @nathan-muluget [here](https://expensify.slack.com/archives/C049HHMV9SM/p1681371354691769?thread_ts=1681292087.572289&cid=C049HHMV9SM)

**Comment by michaelhaxhiu** [comment]
I have a feeling this one is internal. But feel free to change if I'm wrong!

**Comment by MelvinBot** [comment]
Triggered auto assignment to Contributor Plus for review of internal employee PR - @thesahindia (`Internal`)

**Comment by MelvinBot** [comment]
@michaelhaxhiu, @thesahindia Huh... This is 4 days overdue. Who can take care of this?

**Comment by michaelhaxhiu** [comment]
@thesahindia do you agree that this one is `internal`? Or could it be possibly `external`?

**Comment by MelvinBot** [comment]
Triggered auto assignment to @jasperhuangg (`Engineering`), see https://stackoverflow.com/c/expensify/questions/4319 for more details.

**Comment by MelvinBot** [comment]
@michaelhaxhiu @jasperhuangg @thesahindia this issue was created 2 weeks ago. Are we close to a solution? Let's make sure we're treating this as a top priority. Don't hesitate to create a thread in #expensify-open-source to align faster in real time. Thanks!

**Comment by jasperhuangg** [comment]
Agree that it should be internal, will investigate today

**Comment by jasperhuangg** [comment]
Still looking into this, not overdue

**Comment by jasperhuangg** [comment]
I'm struggling to reproduce this because of Pusher errors that are happening due to our latest deploy. I'll take another look when things are back to normal

**Comment by MelvinBot** [comment]
@michaelhaxhiu @jasperhuangg @thesahindia this issue is now 3 weeks old. There is one more week left before this issue <a href='https://stackoverflowteams.com/c/expensify/questions/15233/15234#15234'>breaks WAQ</a>. What needs to happen to get a PR in review this week? Please create a thread in #expensify-open-source to discuss. Thanks!

**Comment by jasperhuangg** [comment]
I'm still holding off on looking at this issue because of the API errors that we've been having on staging and production lately.

**Comment by jasperhuangg** [comment]
Hoping to look into this this week, swamped with other priorities that seem more important than this edge-casey issue.

**Comment by jasperhuangg** [comment]
Still on HOLD

**Comment by michaelhaxhiu** [comment]
@jasperhuangg as a reminder, for `internal` bugs we should ideally have the engineer find a volunteer to take it over if they need to unassign (rather than just unassigning). Did you have a chance to solicit a volunteer internally (e.g. in #engineering-chat)

**Comment by michaelhaxhiu** [comment]
I posted internally here - https://expensify.slack.com/archives/C03TQ48KC/p1685050375117329

**Comment by youssef-lr** [comment]
I'm gonna have time to look into this this week.

**Comment by michaelhaxhiu** [comment]
Fantastic, I'll provide a small nudge if it slips off your radar 🙏

**Comment by michaelhaxhiu** [comment]
Lil nudge :) DM'ed Youssef too.

**Comment by youssef-lr** [comment]
I haven't managed to get to this yet. I will do my best to get to it this week.

**Comment by youssef-lr** [comment]
Currently focused on porting single level control policies to newDot as part of the Teachers Unite project, I will probably have time for this starting next week.

**Comment by youssef-lr** [comment]
Discussing the issue [here](https://expensify.slack.com/archives/C056AV5V8UX/p1688596117850269?thread_ts=1688579188.349419&cid=C056AV5V8UX).

**Comment by youssef-lr** [comment]
Closing as per the discussion above.
